### DISCUSSION TITLE
[Envoy API Gateway] Rate limiter plugin vs endpoints rate limiter

### DISCUSSION THREAD

**Discussion by frengooo** [mia-platform/community#527] [Q&A] [answered]
### What product are you having troubles with?

Console

### What Console version are you using?

v13.0.5

### Description

Hi guys,
I found in the documentation the "Envoy Rate Limiter" plugin available on the Mia Marketplace.
I am not understanding a couple of points:
- is the Rate Limiter plugin mandatory to set the `rate limit` property inside the configurations of a specific enpoint?
<img width="1247" alt="Screenshot 2024-07-26 at 11 57 19" src="https://github.com/user-attachments/assets/f66747a5-0eab-4f41-aca2-558c8c1a5974">

- what are the differences through the Rate Limiter plugin and the property mentioned above?

- how they works together? I mean, the first applied rate limiter is the global one (I think the plugin one) and below the endpoint-specific rate limiters are applied?

Thanks 😄

**Comment by davidebianchi** [comment, marked as answer]
Hi @silversoul93! The rate limit property should work without the need of the plugin. The plugins allows to enable a rate limit shared across different envoy replicas, with different types of possible configurations. 

You can find detailed explanation on the service repository: https://github.com/envoyproxy/ratelimit

**Reply by frengooo** [reply]
Ok clear, I'll take a look on the documentation above.
Thanks @davidebianchi
